mapdata/WarcraftLegacies/UpgradeData/Skin/858804566.json

```diff
@@ -0,0 +1,21 @@
+{
+  "OldId": 1751347026,
+  "NewId": 858804566,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Level": 1,
+      "Id": 829579623,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNSteelRanged.blp"
+    },
+    {
+      "Level": 1,
+      "Id": 1835101799,
+      "Type": 3,
+      "Value": "Quest Completed: Warsong Outpost"
```

Comment:
Research names must match the corresponding quest name precisely. This one doesn't.

Reply:
Good to know. Resolved.